packages/paste-website/src/components/site-wrapper/sidebar/SidebarMobile.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
```

Comment:
thought (blocking): we shouldn't need SidebarDesktop _and_ SidebarMobile anymore since the new Sidebar is responsive